community/demos.md

```diff
@@ -2,6 +2,9 @@ If you have a demo of CloudEvents in action, please add a link here. If there
 isn't an associated blog or github repo, feel free to add descriptive text as a
 markdown file in `community/demos/`.
 
+- [Orchestrating Cloud Events](https://salaboy.com/2020/05/18/orchestrating-cloud-events-with-zeebe/)
+  - Blog post describing how Cloud Events can be orchestrated using an orchestration engine. Example included that you can run in your own environment with links to GitHub repositories needed. 
```

Comment:
minor thing, but can you wrap this at 80 columns to align with the rest of the doc?

Reply:
@duglin thanks a lot for the comment. Is there any way to make sure that it just breaks at 80 chars? I've tried several options but line-breaks will not happen inside bullets.

Reply:
just add a new line and then indent the next line, see https://github.com/cloudevents/spec/pull/646/files#diff-4c22f53fb332a4e6f2c6a9ecd53283aaL17